examples/development/ranking_server.py

```diff
@@ -36,7 +36,7 @@
 
 # Initialize the Grafana Metrics Middleware
 TEAM_ID = "development-shuffle"
-metrics_middleware = GrafanaMetricsMiddleware(app, team_id=TEAM_ID)
+metrics_middleware = GrafanaMetricsMiddleware(app, job_name="ranker", team_id=TEAM_ID)
```

Comment:
Thanks for adding this so we can at least check if we're breaking dev. Now that we're just exposing an endpoint instead of pushing, I'm less concerned with that. But good to keep it here anyway.